migrations/20190820132522_initial.js

```diff
@@ -0,0 +1,32 @@
+exports.up = function(knex) {
+    return Promise.all([
+      knex.schema.createTable("projects", function(table) {
+        table.increments("id").primary();
+        table.string("project_name");
+  
+        table.timestamps(true, true);
+      }),
+  
+      knex.schema.createTable("palettes", function(table) {
+        table.increments("id").primary();
+        table.string("palette_name");
+        table.string("c1");
+        table.string("c2");
+        table.string("c3");
+        table.string("c4");
+        table.string("c5");
+        table.integer("project_id").unsigned();
```

Comment:
I believe you have to leave out the semicolon after unsigned() so that the next line can connect to it

Reply:
Just to reiterate, this does not need to be removed!